packages/avatar/src/__stories__/infoOptions.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import RichText from "@hig/rich-text";
+
+import DefaultExport from "../index";
```

Comment:
Makes for easier copy/pasting. If you can think of a way for this file to be shared, I'm open to it. But it seems like dynamic imports are a no-no with ES6.

Reply:
If we want to strictly use ES modules, then yeah, we'd would have to write some sort of Webpack plugin.

Since this is for our storybook, I'm fine with using `require` if it means we can avoid copy and pasting it.